gensim/models/word2vec.py

```diff
@@ -1180,9 +1180,16 @@ def intersect_word2vec_format(self, fname, lockf=0.0, binary=False, encoding='ut
                         self.wv.syn0[self.wv.vocab[word].index] = weights
         logger.info("merged %d vectors into %s matrix from %s" % (overlap_count, self.wv.syn0.shape, fname))
 
+    def get_words_from_vocab(self):
```

Comment:
For new KeyedVectors functionality, I don't think we need to add forwarding methods to Word2Vec – those exist to ease the way for older code; any new code can access the KeyedVectors methods directly. (This goes for this method and `most_similar_among()`.)